sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -830,6 +830,13 @@ def test_neighbors_badargs():
         assert_raises(ValueError,
                       nbrs.predict,
                       [])
+        if isinstance(cls, neighbors.KNeighborsClassifier) or \
+                isinstance(cls, neighbors.KNeighborsRegressor):
+            nbrs = cls(n_neighbors=-1)
+            assert_raises(ValueError,
+                      nbrs.fit,
+                      X, y)
```

Comment:
style: please use a single line for function calls that fit in 80 columns.

Related: you should run the [pep8](https://pypi.python.org/pypi/pep8) linter to identify code style issues (only consider the errors that match the lines that you edit in your branch).